website/blog/2018-07-27-removing-babels-stage-presets.md

```diff
@@ -180,14 +176,14 @@ This is something that we'd like to continue to do moving forward as another ind
 
 ### Ecosystem Maintenance Burden
 
-A language's "syntax budget" doesn't just apply to the complexity of the language itslef but can extend down to the tooling. Each new syntax can brings [new burdens](http://jshint.com/blog/new-lang-features/) for the maintainers of other JavaScript projects.
+A language's "syntax budget" doesn't just apply to the complexity of the language itself but can extend down to the tooling. Each new syntax addition brings new [burden](http://jshint.com/blog/new-lang-features/) to the maintainers of other JavaScript projects.
 
-Once a new syntax is "added" many things need updating: parsers (`babylon`), syntax highlighting (`language-babel`), linters (`babel-eslint`), test frameworks (jest/ava), formatters (`prettier`), code coverage (`instanbul`), minifiers (`babel-minify`), and more.  There have been many issues brought up on projects like `acorn`, `eslint`, `jshint`, `typescript`, and others to support Stage 0 proposals because they were in Babel. There aren't many projects that would adhere to a policy that required them to support any proposal since that would be extremely demanding to maintain. In many ways, it's surprising we even attempt to handle it in Babel itself given the constant updates and churn.
+Once new syntax is proposed, many things need updating: parsers (`babylon`), syntax highlighting (`language-babel`), linters (`babel-eslint`), test frameworks (jest/ava), formatters (`prettier`), code coverage (`instanbul`), minifiers (`babel-minify`), and more.  There have been many issues brought up on projects like `acorn`, `eslint`, `jshint`, `typescript`, and others to support Stage 0 proposals because they were in Babel. There aren't many projects that would adhere to a policy that required them to support any proposal since that would be extremely demanding to maintain. In many ways, it's surprising we even attempt to handle it in Babel itself given the constant updates and churn.
 
-Who is doing that work, and is it our responsibility to make sure everything works? Every one of those projects (mostly volunteers) is lacking in help in almost every aspect, and yet we continouly get complaints about this across the board. How are we to as a community take responsibility for dealing with these changes (same with open source as a whole)?
+Who is doing that work, and is it our responsibility to make sure everything works? Every one of those projects (mostly volunteers) is lacking in help in almost every aspect, and yet we continually get complaints about this across the board. How are we, as a community, to take responsibility for dealing with these changes (same with open source as a whole)?
```

Comment:
This is a great point, but the section might be stronger with a comment to tie it all up, like this:

> Babel has taken on the unusual burden of supporting these experimental features; at the same time, it's reasonable that other projects take a more conservative policy. If you'd like to see new language features supported across the ecosystem, [contribute to TC39](https://github.com/tc39/ecma262/blob/master/CONTRIBUTING.md) to bring these proposals to Stage 4.